packages/analytics/matomo/usePaq.ts

```diff
@@ -1,6 +1,14 @@
+declare global {
+   interface Window {
+      _paq: any;
```

Comment:
I think it should be `{push: (args: array) => void}` or however it's expressed in typescript because we use it like `_paq.push([...])`. Does this make sense?

Reply:
I think so. Because either it's an array with `push` or because it's the initialized object from Matomo that has `push` defined on it.

Reply:
Are you saying I can put `{push: (args: array) => void}` for the type?